README.md

```diff
@@ -93,30 +93,30 @@ qemu_path_image=$HOME/p/virty.qcow2
 ```
 
 3) Done, every time you execute any command in the root dir of drm-work the
-configuration file will be read. If file does not exists, the default operation
+configuration file will be read. If the file does not exist, the default operation
 will be used.
 
 # How to
 
-> Help commands:
+> Get help or list existing commands:
 
 ```
 kw help|h
 ```
 
-> Build a Kernel and install in the Qemu image:
+> Build a Kernel and install it in the Qemu image:
 
 ```
 kw bi
 ```
 
-> Mount Qemu image to transfer data:
+> Mount a Qemu image to transfer data:
```

Comment:
IMHO, we should say "the" instead of "a" since we are talking about the specific Qemu image defined in the kworkflow.config file (global or local). Maybe we should indeed add this information here, to make it cleaner...

Reply:
Yeah, I don't know the tool, so it's difficult to tell for me.